health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/utils/appConfigHelpers.js

```
@@ -254,23 +254,23 @@
       const typeAndFormat = getTypeAndFormatFromAppType(field, fieldTypeMasterData);
       const toArrayFields = addToArrayFields(field, fieldTypeMasterData); // TODO @nabeel @jagan right now this works for only validation array, we should think to expose to change the main config dynamically 
       return {
-        label: field.label || "",
+        label: field?.label || "",
         order: fieldIndex + 1,
-        value: field.value || "",
+        value: field?.value || "",
         hidden: false, // can't be derived from updatedData unless explicitly added
```

Biome (1.9.4)

[error] This property value named hidden is later overwritten by an object member with the same name.

Overwritten with this value.

If an object property with the same name is defined multiple times (except when combining a getter with a setter), only the last definition makes it into the object and previous definitions are ignored.
Unsafe fix: Remove this property value named hidden


(lint/suspicious/noDuplicateObjectKeys)